static/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -65,6 +67,14 @@ class TeamMembers extends AsyncView<Props, State> {
   componentDidMount() {
     // Initialize "add member" dropdown with data
     this.fetchMembersRequest('');
+
+    const {teamId} = this.props.params;
+    const team = TeamStore.getBySlug(teamId);
+    if (team) {
+      this.setState({
+        team,
+      });
```

Comment:
Can you check if this is actually needed? 

It seems like `this.props.team` is populated for this AsyncView because it is passed in by `<TeamDetails/>`, so it might make sense to use that instead? You'll need to update the prop typing.

Reply:
it looks like none of the fetching the team from the TeamStore stuff is necessary since the team is already a prop